drivers/mmc/host/Kconfig

```diff
@@ -35,6 +35,15 @@ config JZMMC_V12_MMC0_PB_4BIT
 	bool "GPIO B, data with 4 bit"
 endchoice
 
+config JZMMC_V12_MMC0_1BIT
+    bool "Restrict host controller to 1 bit bus width"
+    default n
+    help
```

Comment:
should we add depends on CONFIG_SOC_T23 to restrict this option? thoughts @themactep ?

Reply:
If you believe this is specific to the T23 soc. But I think that any soc can be on a board that only uses 1bit bus. So we can leave it as is since it is off by default anyway. And you can drop `default n` since it is n by default.